src/models/comment.js

```diff
@@ -0,0 +1,22 @@
+//Requires Mongoose
+const mongoose = require("mongoose");
+
+//Creates variables to store Schema
+
+const Schema = mongoose.Schema;
+
+//Creates comment schema
+
+const commentSchema = Schema({
+  userID: { type: Schema.Types.ObjectId, ref: "User" },
+  username: { type: Schema.Types.ObjectId, ref: "User" },
```

Comment:
I would just do "username: String" instead, because you already did it inside the userID, no need at this point.